### ISSUE TITLE
high contrast mode displays black

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-visua11y#99]
### Subject of the issue
When enabling _high contrast_ mode, the course displays black (text and background) so none of the content is visible. 

Visua11y drawer open:
<img width="1099" alt="visually_open" src="https://github.com/cgkineo/adapt-visua11y/assets/7045330/0cd3997d-3868-4c0d-927c-71010e2a6409">

When _high contrast_ is enabled:
<img width="1099" alt="with_high_contrast_enabled" src="https://github.com/cgkineo/adapt-visua11y/assets/7045330/7a6acf27-607f-4e65-885a-16bc74f142f9">

I can replicate with both Drawer and Notify display.

### Your environment
I can replicate this in the following (exports from AATs):
*FW 5.31.23 / FW 5.31.27
*Core 6.39.1 / Core 6.39.5
*Vanilla 9.8.4
*Visually 2.5.2

Issue doesn't occur in latest Adapt and works as expected.

### Steps to reproduce
Install and configure Visually as per [example.json](https://github.com/cgkineo/adapt-visua11y/blob/master/example.json). Go to Visually settings and enable _high contrast_.

### Expected behaviour
High contrast should increase the contrast between foreground (text) and background. e.g the course background white and text black, darken drawer background and text remains white.

**Comment by oliverfoster** [comment]
"_highContrastLuminanceThreshold" is a string when it should be a number
